### PR TITLE
Statsd 6.3.1.0

### PR DESCRIPTION
**Pull request by shekhar8617**
upstream PR  for  statsd 6.3.1.0

### CODE REVIEW

**Comment by shekhar8617** [comment]
Hi @mpereira  is there any update on PR https://github.com/Automattic/elasticsearch-statsd-plugin/pull/29 when will this merge to master

**Comment by anandnalya** [comment]
Hi @shekhar8617,

This PR is not running any of the unit tests. [Travis CI Logs](https://travis-ci.org/Automattic/elasticsearch-statsd-plugin/builds/418116411#L1484)

File: pom.xml [hunk 1]
```diff
@@ -106,7 +106,13 @@
...
             <scope>test</scope>
         </dependency>
-    </dependencies>
+	<dependency>
+           <groupId>org.testng</groupId>
+           <artifactId>testng</artifactId>
+           <version>6.8.8</version>
+           <scope>test</scope>
+       </dependency>
+   	 </dependencies>
```
Comment (mpereira) [inline pom.xml]: Why is this change (and the two below) necessary?

Comment (shekhar8617) [reply pom.xml]: Hi @mpereira  please update

**Comment by anandnalya** [comment]
[Release](https://github.com/Automattic/elasticsearch-statsd-plugin/releases/tag/6.3.1.0) for ES 6.3.1 is available now.

File: pom.xml (hunk 1 above)
Comment (shekhar8617) [reply pom.xml]: Hi @mpereira https://github.com/Automattic/elasticsearch-statsd-plugin/pull/29 has merge to master please merge statsd_6.3.1 in mesosphere/elasticsearch-statsd-plugin master

**Pull request closed without merging**